packages/xo-web/src/common/intl/messages.js

```diff
@@ -257,7 +257,12 @@ const messages = {
   homeMissingPatches: 'Missing patches',
   homePoolMaster: 'Master:',
   homeResourceSet: 'Resource set: {resourceSet}',
+  haltedItem: 'Halted {type}',
   highAvailability: 'High Availability',
+  pausedItem: 'Paused {type}',
+  powerState: 'Power state',
+  runningItem: 'Running {type}',
+  suspendedItem: 'Suspended {type}',
```

Comment:
I don't think we need to pass `type`:
```js
runningVms: 'Running VMs',
haltedVms: 'Halted VMs',
pausedVms: 'Paused VMs',
suspendedVms: 'Suspended VMs',
runningHosts: 'Running hosts',
haltedHosts: 'Halted hosts',
```
That's only 2 more messages and it would be easier to translate.